src/plugins/dataflow/model/utilities/node.ts

```diff
@@ -172,6 +172,30 @@ export const NodeOperationTypes = [
     numberSentence: (n1: string, n2: string) => `ceil(${n1}) = `,
     icon: EmgIcon
   },
+  {
+    name: "Ramp",
+    type: "transform",
+    method: (n1: number, n2: number, tMinus1?: number) => {
```

Comment:
`tMinus1` is not a very intuitive name. If I'm reading the code correctly, wouldn't something like `prevValue` be appropriate?